airflow/models.py

```diff
@@ -3024,6 +3024,24 @@ def get_active_runs(self, session=None):
         return active_dates
 
     @provide_session
+    def count_active_runs(self, external_trigger=None, session=None):
```

Comment:
Got a comment from @bolkedebruin that this would increase the scope of this PR, so punting to other refactor.